tools/src/main/java/com/nedap/archie/flattener/CAttributeFlattener.java

```diff
@@ -174,43 +173,64 @@ private void mergeObjectIntoAttribute(CAttribute parentAttribute, CObject specia
      *
      * after[id3]
      * ELEMENT[id2]
+     * ELEMENT[id0.1]
+     * before[id4]
      * ELEMENT[id3.1]
      *
-     * Reorder it and remove sibling orders:
+     * Where id3.1 itself is inside a block reordered with sibling nodes. So, reorder it and remove sibling orders:
      *
      * ELEMENT[id3.1]
```

Comment:
the before[id4] in this example is not removed, perhaps that should be added to the comments here